userguide/cli.rst

```diff
@@ -1051,3 +1051,25 @@ To view the BIOS report, type the command with no arguments:
 
 `dmidecode(8) <https://linux.die.net/man/8/dmidecode>`__
 describes the supported strings and types.
+
+
+.. index:: Midnight Commander
+.. _Midnight_Commander:
+
+Midnight Commander
+------------------
+
+Midnight Commander is a program used to manage files from the shell.
+Open the application by running the command :command:`mc`.
+The arrow keys are used to navigate and select files. The
+function keys are used to perform operations such as renaming, editing
+and copying files. These resources provide more information about using
+mc:
+
+* `Midnight Commander wikipedia page <https://en.wikipedia.org/wiki/Midnight_Commander>`__
+
+* `Midnight Commander website <https://midnight-commander.org/>`__
+
+* `mc(1) <https://linux.die.net/man/1/mc>`__
```

Comment:
Use the FreeBSD man page when possible: https://www.freebsd.org/cgi/man.cgi?query=mc